src/ModelCleanedEvent.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Spatie\ModelCleanup;
+
+use App\Events\Event;
+use Illuminate\Queue\SerializesModels;
+
+class ModelCleanedEvent extends Event
```

Comment:
Extending `Event` is not necessary. Removing this will probably make the tests pass.